src/platform/intel/cavs/lib/clk.c

```diff
@@ -80,16 +88,10 @@ static inline int get_cpu_current_freq_idx(void)
 	return clk_info->current_freq_idx;
 }
 
-static inline void set_cpu_current_freq_idx(int freq_idx)
+static inline void set_cpu_current_freq_idx(int freq_idx, bool release_unused)
```

Comment:
Need a brief comment that this function is supposed to be called by public API implementations, so that internal switching does not override active_freq_idx.

Reply:
added